reorder_python_imports.py

```diff
@@ -769,7 +772,7 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     )
     group = parser.add_mutually_exclusive_group(required=False)
     group.add_argument(
-        '--diff-only', action='store_true',
+        '--diff-only', '--diff', action='store_true',
```

Comment:
argparse has prefix matching so you don't need to add this option here (in fact this will *break* someone using prefix matching because the options become ambiguous)